packages/blocks/image/src/components/media-with-caption.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { VFC } from "react";
+import React, { forwardRef, ForwardRefRenderFunction } from "react";
```

Comment:
I think you can undo these changes and the equivalent in `video` since a ref is no longer forwarded to this component.